apps/web/src/components/layout/components/ListPage.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled';
+import { IPaginationProps, PageContainer, Pagination, colors } from '@novu/design-system';
+import PageHeader from './PageHeader';
+import { ITableProps } from '@novu/design-system/dist/types/table/Table';
+import { ComponentProps, PropsWithChildren } from 'react';
+
+// values directly from designs -- see comment below about why they're necessary
+const FOOTER_FADE_OVERLAY_HEIGHT_PX = 20;
+const FOOTER_HEIGHT_PX = 64;
+const TOTAL_FOOTER_HEIGHT_PX = FOOTER_HEIGHT_PX + FOOTER_FADE_OVERLAY_HEIGHT_PX;
+
+const StickyFooter = styled.footer`
+  position: sticky;
+  bottom: 0;
+  /**
+   * :( Required CSS witchery because the PageContainer and other contained elements doesn't use Flexbox or Grid.
+   * We can't use position: fixed or absolute because the PageContainer width is dynamic, and we can't inherit it.
+   * 'sticky' alone doesn't work, so we have to force the element to a calculated position.
+   */
+  top: calc(100% - ${TOTAL_FOOTER_HEIGHT_PX}px);
```

Comment:
In this case, I did actually mean width because having a dynamic width prevents us from being able to use `position: fixed` with `width: 100%` since it tries to use 100% of the viewport instead of 100% of the container